src/compas_timber/assembly/assembly.py

```diff
@@ -201,3 +201,37 @@ def set_topologies(self, topologies):
     @property
     def topologies(self):
         return self._topologies
+
+    @property
+    def center_of_mass(self):
+        """Returns the center of mass of the assembly.
+
+        Returns
+        -------
+        list
```

Comment:
should be a `Point`, no?

Reply:
yeah I should have checked what copilot wrote....

fixed it now